### ISSUE TITLE
Shows, Casino and Hotel Pages

### ISSUE THREAD

**Issue by amol-anand** [hlxsites/caesars#34]
[/caesars-palace/hotel](https://www.caesars.com/caesars-palace/hotel)
[/caesars-palace/hotel/wifi](https://www.caesars.com/caesars-palace/hotel/wifi)
[/caesars-palace/hotel/deals](https://www.caesars.com/caesars-palace/hotel/deals)
[/caesars-palace/casino](https://www.caesars.com/caesars-palace/casino)
[/caesars-palace/casino/race-and-sports-book](https://www.caesars.com/caesars-palace/casino/race-and-sports-book)
[/caesars-palace/shows](https://www.caesars.com/caesars-palace/shows)
[/caesars-palace/shows/the-colosseum](https://www.caesars.com/caesars-palace/shows/the-colosseum)
[/caesars-palace/shows/absinthe](https://www.caesars.com/caesars-palace/shows/absinthe)
[/caesars-palace/shows/rod-stewart](https://www.caesars.com/caesars-palace/shows/rod-stewart)
[/caesars-palace/shows/sting](https://www.caesars.com/caesars-palace/shows/sting)
[/caesars-palace/shows/morrissey](https://www.caesars.com/caesars-palace/shows/morrissey)
[/caesars-palace/shows/garth-brooks](https://www.caesars.com/caesars-palace/shows/garth-brooks)

@tmathern Update: [/caesars-palace/hotel/caesars-palace-concierge](https://www.caesars.com/caesars-palace/hotel/caesars-palace-concierge) moved to its own issue: https://github.com/hlxsites/caesars/issues/69

**Comment by tmathern** [comment]
https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/casino

**Comment by amol-anand** [comment]
The product details pages are dependent on #164 and content migration of that data into excel files
